tests/test_profile_data.py

```diff
@@ -176,3 +176,23 @@ def test_profile_using_spark(df_spark):
     )
 
     assert len(generate_raw_profile_dataset([["first_name", "blank"]], linker)) == 0
+
+
+def test_profile_null_columns(caplog):
+
+    df = pd.DataFrame(
+        [
+            {"unique_id": 1, "test_1": 1, "test_2": None},
+        ]
+    )
+
+    linker = DuckDBLinker(df)
+
+    linker.profile_columns(["test_1", "test_2"])
```

Comment:
Can you add in a `concat` test here too:
```suggestion
    linker.profile_columns(["test_1", "test_2", ["test_1", "test_2"]])
```

Reply:
and then updated the logcap check...

Reply:
Done